lib/libshare/nfs.c

```diff
@@ -29,12 +29,18 @@
 #include <strings.h>
 #include <fcntl.h>
 #include <sys/wait.h>
+#include <sys/list.h>
 #include <unistd.h>
 #include <libzfs.h>
 #include <libshare.h>
 #include "libshare_impl.h"
 
+#if !defined(offsetof)
+#define	offsetof(s, m)  ((size_t)(&(((s *)0)->m)))
+#endif
```

Comment:
Why was this needed, `offsetof` is part of stddef.h
http://man7.org/linux/man-pages/man3/offsetof.3.html